src/content/docs/new-relic-solutions/new-relic-one/new-navigation-transition-guide.mdx

```diff
@@ -19,61 +19,91 @@ import transitionGuideUserMenu from 'images/transition-guide_screenshot-full_use
 * **The left nav is stable.** No matter how often you click around within a capability or a feature, the left nav is a steady and stable place for you to return to in the UI.
 * **Work in the dark.** Our designers have refined and crafted our dark mode into an elegant experience for those of you who like working in the dark. 
 
-## Turn it on (or off) [#on]
+## Make New Relic work for you [#make-new-relic]
+
+<img
+  title="The all capabilities page with pinned items"
+  alt="A screenshot of the all capabilities page with pins highlighted."
+  src={transitionGuideAllCapabilities}
+/>
+
+Dealing with increasing complexity across your stack requires intuitive access to telemetry data. You can now customize New Relic for easy access to the tools you use every day. 
+
+Go to **All capabilities** to see a catalog of all of our tools, views, and features. Click around to discover and learn more about things you aren't using and to try out new solutions to your problems.
 
-To turn on the new navigation: Click your user menu in the top right corner, and then toggle **New design** to **On**. (If you need to turn it off for any reason, click the user menu in the bottom left corner of the new left nav, then click the toggle to **Off**.)
+Do you use something all the time? Click its pin to stick it the left nav for easy access. Click an active pin to remove that capability from the left nav.
 
-Thank you for opting in to try it out. Let us know what you think about our new UI!
+But, before you start, here are our recommendations for how to begin customizing your New Relic experience.
 
-## How this serves you better [#why]
+### Pin the query builder for easy access to your data  [#query-builder]
 
 <img
-  title="The new home UI"
-  alt="A screenshot of the new home page UI."
-  src={transitionGuideTransitionGuide}
-  style={{ maxWidth: '70%' }}
+  title="Query builder"
+  alt="A gif demonstrating how to use the query builder in the New Relic UI. "
+  src={transitionGuideQueryBuilderUiRedesign}
 />
 
-With this update you have a simpler, more stable UI experience that gets out of your way and creates more screen real estate so you can focus on what's most important: the health of your system.
+<figcaption>
+  Pin the query builder to the left navigation to quickly access your data. 
+</figcaption>
+
+Before you dig into making New Relic your own, the first thing we recommend is that you go to **All capabilities**  and click the pin next to "Query your data." This will permanently stick the query builder to the left navigation.  Our new [query builder](/docs/query-your-data/explore-query-data/query-builder/introduction-query-builder/) is an essential piece of the New Relic ecosystem that allows you an easy way to filter, organize, and understand your data. 
```

Comment:
probably could just delete that last sentence going 'is an essential piece' and link from 'query builder' to its doc at first mention. we explain below what it does so I wouldn't belabor the point too much; most existing customers know what the query builder is.